manifests/templates/disks-images-provider.yaml.in

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: host-path-disk-datavolume1
+  labels:
+    kubevirt.io: ""
+    os: "datavolume1"
```

Comment:
what is the `os` label for?

Reply:
that's now we match a PVC with the PV. we're using a selector called "os".  I pulled this logic from KubeVirt where "os" made more sense in their context than it does in CDI.

Reply:
Do we really need to create PVs any more? We should have 10 local volume PVs available from the kubevirtci image at this point that we can leverage.

Reply:
@awels some dev documentation would be helpful there. Something that just outlines what providers contain local storage support now, and an example spec for how to construct a PVC to ultilize local storage provider.

Reply:
@davidvossel https://github.com/kubevirt/kubevirtci/pull/35 for the documentation.

Reply:
@awels 
> We should have 10 local volume PVs available from the kubevirtci image at this point that we can leverage.

Is the implication here that functional tests will be closely coupled w/ the kubevirtCI env by expecting PVs to exist already?

Reply:
Not any more than us expecting a kubernetes environment to exist, I don't think its unreasonable for our tests to expect some kind of PV provider exists in the same environment, as these are CDI related tests.

Reply:
@awels awesome, thanks! I updated this PR to use local storage now. works great, this is super useful. I'll be using it in KubeVirt soon as well

Reply:
@awels wrote:
>  I don't think its unreasonable for our tests to expect some kind of PV provider exists in the same environment

My background for e2e is in k8s. In that env the test ensures that the storage class, or pv or whatever storage is needed is explicitly created in the test setup (BeforeEach is common). If I understand what's going on here correctly, the test image is providing the pvs. If true that is quite different from k8s. Again, if true, what is the motivation for doing storage this way?

Reply:
For k8s e2e it would make sense for the test to create storage/whatever resources is needed for the test. As the test is exercising the framework that is using that particular resource. 

In our case it is different, we are testing something that is running on top of k8s, and expects a working k8s environment. Typically in a working k8s environment there is some kind of storage.

The motivation for doing it like this? We found that everything that was using the kubevirtci image needed some kind of storage, and everyone was doing something different (most of the time a hostPath based PV). So instead of having everyone reinvent the wheel each time, we implemented some basic local volume based storage in the image. Now everyone has some default storage available.